librz/util/graph_drawable.c

```diff
@@ -56,11 +56,13 @@ RZ_API RZ_OWN char *rz_graph_drawable_to_dot(RZ_NONNULL RzGraph /*<RzGraphNodeIn
 
 	rz_list_foreach (nodes, it, node) {
 		RzGraphNodeInfo *print_node = (RzGraphNodeInfo *)node->data;
-		const char *body = print_node->body;
+		char *body = print_node->body;
+
 		if (!body || !*body) {
 			rz_strbuf_appendf(&buf, "%d [URL=\"%s\", color=\"lightgray\", label=\"%s\"]\n",
 				node->idx, print_node->title, print_node->title);
 		} else {
+			body = rz_str_replace(body, "\"", "\\\"", 1);
 			rz_strbuf_appendf(&buf, "%d [URL=\"%s\", color=\"lightgray\", label=\"%s\\n%s\"]\n",
 				node->idx, print_node->title, print_node->title, body);
 		}
```

Comment:
The double free issue is very simple, `print_node->body` owns the buffer, then since you replace a shorter string with a longer string, the old pointer gets freed and a new pointer is allocated.
then the old pointer is never updated.
Essentially you are taking the ownership of something that you do not own.
I suggest that you do
```c
char *body = strdup(print_node->body);
....
} else {
	body = rz_str_replace(body, "\"", "\\\"", 1);
	rz_strbuf_appendf(&buf, "%d [URL=\"%s\", color=\"lightgray\", label=\"%s\\n%s\"]\n",
				node->idx, print_node->title, print_node->title, body);
	free(body);
}
```